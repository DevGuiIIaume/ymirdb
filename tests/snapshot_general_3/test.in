set a 1
set b a 2
snapshot
pop a
snapshot
del b
snapshot
del a
snapshot
checkout 1
list entries
checkout 2
list entries
forward a
forward b
backward a
backward b
sum a
sum b
len a
len b
checkout 3
list entries
sum a
len a
forward a
forward b
checkout 4
list entries
bye
