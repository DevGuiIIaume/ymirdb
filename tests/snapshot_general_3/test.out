> ok

> ok

> saved as snapshot 1

> 1

> saved as snapshot 2

> ok

> saved as snapshot 3

> ok

> saved as snapshot 4

> ok

> b [a 2]
a [1]

> ok

> b [a 2]
a []

> nil

> a

> b

> nil

> 0

> 2

> 0

> 1

> ok

> a []

> 0

> 0

> nil

> no such key

> ok

> no entries

> bye
